qemu/tests/qemu_io_blkdebug.py

```diff
@@ -94,8 +95,8 @@ def run(test, params, env):
             try:
                 image_io.snapshot_del(blkdebug_cfg=blkdebug_cfg)
                 output = ""
-            except error.CmdError, err:
-                output = err.result_obj.stderr
+            except process.CmdError, err:
```

Comment:
exceptions.TestError() raised means that snapshot is created failed. we don't need to catch this exception.